### PR TITLE
bpfman release v0.5.3

### PR DESCRIPTION
**Pull request by anfredette**

### CODE REVIEW

File: changelogs/CHANGELOG-v0.5.3.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+The v0.5.3 release is a patch release made to fix incorrect image URLs in the
+bpfman-operator bundle.  There are no functional changes to bpfman between
+v0.5.2 and 0.5.3.
```
Comment (Billy99) [inline changelogs/CHANGELOG-v0.5.3.md:3]:
```suggestion
v0.5.2 and v0.5.3.
```

File: examples/config/v0.5.3-selinux/go-app-counter/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Patch the bytecode.yaml to change tag on the "url" field (which is an
+# image) to new value. This actually overwrites the image with the same value.
+patches:
+  - target:
+      kind: BpfApplication
+      name: app-counter
+    patch: |-
+      - op: replace
+        path: "/spec/bytecode/image/url"
+        value: quay.io/bpfman-bytecode/go-app-counter:vv0.5.3
```
Comment (Billy99) [inline examples/config/v0.5.3-selinux/go-app-counter/kustomization.yaml:13]:
```suggestion
        value: quay.io/bpfman-bytecode/go-app-counter:v0.5.3
```
Global change, this needs to be changed everywhere.

File: changelogs/CHANGELOG-v0.5.3.md (hunk 1 above)
Comment (anfredette) [reply changelogs/CHANGELOG-v0.5.3.md]: Done.

File: examples/config/v0.5.3-selinux/go-app-counter/kustomization.yaml (hunk 2 above)
Comment (anfredette) [reply examples/config/v0.5.3-selinux/go-app-counter/kustomization.yaml]: Done.

**Pull request merged**